src/main/java/com/google/cloud/spanner/pgadapter/ProxyServer.java

```diff
@@ -137,17 +137,6 @@ public void stopServer() {
     awaitTerminated();
   }
 
-  public void run() {
```

Comment:
Not sure why this change is in this PR.

Reply:
Oh, this was more of a general cleanup PR, and I noticed that this method is not used anymore.